client/pages/apply.jsx

```diff
@@ -93,10 +94,10 @@ class Apply extends React.Component {
                 type="text"
                 required
                 defaultValue={oneLineIntro}
-                pattern="[A-Za-z0-9,!?\x27\x22\s\.]{8,140}"
+                pattern="[A-Za-z0-9,!?_\-\x27\x22\s\.]{8,140}"
```

Comment:
Anything against just using `\w`?

Reply:
Just laziness. Iirc \w doesn't apply to hyphens, so it's a difference of adding three characters "_\-" vs deleting a bunch of stuff and then adding four chars "\w\-".

Reply:
Next commit has it changed.